### PR TITLE
Update README, instructions, and setup script for Llama 3.1 70B on LoudBox for v0 release

### PR DESCRIPTION
**Pull request by milank94**
### What's changed

- Updated root README to modify LLM table to include vLLM release version.
- Updated Llama 3.1 70B on LoudBox README with instructions for running vLLM server.
- Add "one-line" script for new users to run Llama 3.1 70B + vLLM server demo from end-to-end including build environment, setup weights, and launch server.

### Checklist
- [x] Post commit CI passes

### CODE REVIEW

**Review by cglagovichTT — APPROVED** [review]

**Review by kpaigwar — APPROVED** [review]

**Review by mtairum — APPROVED** [review]

**Comment by mtairum** [comment]
I would add a mention in `models/demos/t3000/llama3_70b` README,  on what to change to make `setup_llama3.sh` work. 

Looking at the script the llama3-70b weights location doesn't seem to be parametrizable, and the user will likely want to change.

If this is intended for power users, that's ok, but at least a comment on the shell script mentioning where to change the location.

**Comment by mtairum** [comment]
Also followed the manual tutorial and it works fine (already had the 70b weights).

File: README.md [hunk 1]
```diff
@@ -21,29 +21,34 @@
...
-| [Mixtral7Bx8 (TP=8)](./models/demos/t3000/mixtral8x7b)        | 32    | [QuietBox](https://tenstorrent.com/hardware/tt-quietbox) | 230      | 14.6  | 33              | 467.2  | [v0.53.0-rc44](https://github.com/tenstorrent/tt-metal/tree/v0.53.0-rc44) |
-| [Falcon7B (DP=32)](./models/demos/tg/falcon7b)                | 1024  | [Galaxy](https://tenstorrent.com/hardware/galaxy)        | 242      | 4.4   | 26              | 4505.6 | [v0.53.0-rc33](https://github.com/tenstorrent/tt-metal/tree/v0.53.0-rc33) |
-| [LLaMA-3.1-70B (DP=4, TP=8)](./models/demos/t3000/llama3_70b) | 128   | [Galaxy](https://tenstorrent.com/hardware/galaxy)        | 190      | 14.3  | 20              | 1835.5 | [v0.52.0-rc31](https://github.com/tenstorrent/tt-metal/tree/v0.52.0-rc31) |
-> **Last Update:** December 2, 2024
 
+| Model                                                         | Batch | Hardware                                                 | ttft (ms) | t/s/u | Target<br>t/s/u | t/s    | TT-Metalium Release                                            | vLLM Release                                                                                      |
```
Comment (uaydonat) [inline README.md:25]: vLLM is not official vLLM, maybe we call it "vLLM Tenstorrent Repo Release"?

Comment (milank94) [reply README.md]: Done. See https://github.com/tenstorrent/tt-metal/pull/15820/commits/18cdd1f2961a0bb59675422feabc7c80c8eb0503

**Comment by milank94** [comment]
> I would add a mention in `models/demos/t3000/llama3_70b` README, on what to change to make `setup_llama3.sh` work.
> 
> Looking at the script the llama3-70b weights location doesn't seem to be parametrizable, and the user will likely want to change.
> 
> If this is intended for power users, that's ok, but at least a comment on the shell script mentioning where to change the location.

@mtairum [Updated](https://github.com/tenstorrent/tt-metal/pull/15820/commits/18cdd1f2961a0bb59675422feabc7c80c8eb0503) it to be a generic script that now takes in the model type from:
- llama-3.1-70b-instruct
- llama-3.1-70b
- llama-3.1-8b-instruct
 - llama-3.1-8b
- llama-3-70b-instruct
- llama-3-70b
 - llama-3-8b-instruct
- llama-3-8b

We can use this as a generic script fro the 3.2 and 3.3 models as well.

**Pull request merged**